packages/server/src/api/routes/tests/viewV2.spec.ts

```diff
@@ -177,6 +177,9 @@ describe.each([
               visible: true,
             },
           },
+          uiMetadata: {
+            foo: "bar",
+          },
```

Comment:
You probably want at least one assertion somewhere to make sure we do actually return this when fetching views from the API.

Reply:
That's what the update test does already isn't it? It's asserting that the response from the update call matches the payload sent up, which includes the `uiMetadata` field that should have been saved.

Reply:
Oh I completely missed that, sorry!